src/screens/Signup/StepInfo/index.tsx

```diff
@@ -99,24 +83,50 @@ export function StepInfo({
       return dispatch({
         type: 'setError',
         value: _(msg`Please enter your invite code.`),
+        field: 'invite-code',
       })
     }
     if (!email) {
       return dispatch({
         type: 'setError',
         value: _(msg`Please enter your email.`),
+        field: 'email',
       })
     }
     if (!EmailValidator.validate(email)) {
       return dispatch({
         type: 'setError',
         value: _(msg`Your email appears to be invalid.`),
+        field: 'email',
       })
     }
+    if (emailChanged && tldtsRef.current) {
+      if (isEmailMaybeInvalid(email, tldtsRef.current)) {
+        prevEmailValueRef.current = email
+        setHasWarnedEmail(true)
+        return dispatch({
+          type: 'setError',
+          value: _(
+            msg`It looks like you may have entered your email address incorrectly. Are you sure it's right?`,
```

Comment:
very minor nit and im sure we're breaking this elsewhere, but we should avoid contractions in the ui - particularly around any account management stuff.

```suggestion
            msg`It looks like you may have entered your email address incorrectly. Are you sure it is correct?`,
```

Reply:
Any change in a string will need to be re-translated, but as I explained to @mozzius in another PR, that's not really a big deal imo, we translators are used to it. And with the translation memory feature that we have with Crowdin now, it's even easier and quicker 💫  So fwiw, my personal suggestion would be to always get the English string as good as it can be, and let us translators worry about keeping up :)

But I did also want to chime in with an alternative suggestion... what about rephrasing it as a request rather than a question?

E.g. something like:

> Please double-check that you've entered your email address correctly.

Reply:
Thanks for that explainer! Also yea, I'm also fine with that suggestion so long as we drop the contraction. Again, I think we've been pretty lose with that in the past but particularly around the sensitive bits of the app/ecosystem I think we should strive to avoid informal writing.

Reply:
Oops, I didn't even realise that I'd included a contraction in my suggestion, d'oh 🤦🏻‍♂️😅

Reply:
On that point, the button has the text `It's correct`, I'm not sure what would work better without a contraction there (`It is correct` somehow doesn't seem quite right)
![IMG_7106](https://github.com/user-attachments/assets/7610b164-8315-4fb2-8b82-e952bd9a6ca1)

Reply:
(I am not a UI expert :upside_down_face:) I think in the actual buttons where we're dealing with space concerns or very brief statements it's fine? It's mostly just other copy around the app - i.e. small paragraphs, sentences, etc. - that we should be a bit more careful. The "It's Correct" seems right here, I agree. And also cannot come up with something better.

Reply:
Alternative would be "Confim email" I guess but I'm ok with "It's correct"